src/test/java/com/microsoft/sqlserver/jdbc/datatypes/TVPWithSqlVariantTest.java

```diff
@@ -353,8 +353,10 @@ public void testDateTime() throws SQLException {
      * 
      * @throws SQLException
      * @throws SQLTimeoutException
+     * https://msdn.microsoft.com/en-ca/library/dd303302.aspx?f=255&MSPPError=-2147217396
+     * Data types cannot be NULL when inside a sql_variant
      */
-    @Test // TODO We need to check this later. Right now sending null with TVP is not supported
```

Comment:
This TODO must stay, as there's a driver bug that throws exception for NULL value. Instead of validating Exception, we should be ideally validating stored NULL value.